control_panel_api/filters.py

```diff
@@ -56,17 +56,11 @@ class UserFilter(DjangoFilterBackend):
     """
     Filter to get visible users.
 
-    Currently superusers see everything, normal users see only themselves,
-    unauthenticated users see nothing.
+    Everyone can see all users.
     """
 
     def filter_queryset(self, request, queryset, view):
-        queryset = super().filter_queryset(request, queryset, view)
-
-        if is_superuser(request.user):
-            return queryset
-
-        return queryset.filter(pk=request.user.pk)
+        return super().filter_queryset(request, queryset, view)
```

Comment:
This makes this class a no-op. Probably it should be removed and the reference to it from `views.py` should be removed.

Reply:
Let me have a look, can't remember what is the default filter class in the views.

Reply:
FYI: Got rid of that class and using the parent class directly now https://github.com/ministryofjustice/analytics-platform-control-panel/pull/113/commits/c1a1230c17a00c1d41b7ac524b849f539002969d